htdocs/admin/pdf.php

```diff
@@ -166,6 +166,14 @@
 		dolibarr_set_const($db, "PDF_SHOW_LINK_TO_ONLINE_PAYMENT", GETPOST('PDF_SHOW_LINK_TO_ONLINE_PAYMENT', 'alpha'), 'chaine', 0, '', $conf->entity);
 	}
 
+	if (GETPOSTISSET('PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME')) {
```

Comment:
No more need of this option.
We have now PDF_THIRDPARTY_NAME_TO_SHOW_IS that manage all cases.

Reply:
PDF_THIRDPARTY_NAME_TO_SHOW_IS is traduct key
PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME is option for backward compatibility